system-tests/docker-compose.yml

```
@@ -152,9 +152,16 @@ services:
       WEB_HTTP_AUTHORITY_PORT: 8184
       WEB_HTTP_AUTHORITY_PATH: /authority
       EDC_IAM_DID_WEB_USE_HTTPS: "false"
+      EDC_CONNECTOR_NAME: registration-service
+      EDC_IDENTITY_DID_URL: did:web:did-server:registration-service
+      EDC_VAULT: /resources/vault/registration-service/registration-service-vault.properties
+      EDC_KEYSTORE: /resources/vault/registration-service/registration-service-keystore.jks
+      EDC_KEYSTORE_PASSWORD: test123
```

nit: error verbose may also be added
```suggestion
      EDC_KEYSTORE_PASSWORD: test123
      EDC_ERROR_RESPONSE_VERBOSE: "true"
```